src/common/cert_vfy.c

```diff
@@ -408,7 +408,7 @@ static X509_STORE * setup_store(cert_policy *policy) {
     }
   }
   /* add needed hash dir pathname entries */
-  if ( (policy->ca_policy) && (is_dir(policy->ca_dir)>0) ) {
+  if ( (!policy->no_ca_policy==0) && (is_dir(policy->ca_dir)>0) ) {
```

Comment:
I think this now changes the meaning and should be as initially suggested in the previous comment:
```suggestion
  if ( (policy->no_ca_policy==0) && (is_dir(policy->ca_dir)>0) ) {
```